TAO/TAO_IDL/util/utl_global.cpp

```diff
@@ -1953,7 +1953,7 @@ namespace
   {
   public:
     explicit OldState (bool disable_output = false)
-        : old_filename_ (idl_global->filename ()),
+        : old_filename_ (idl_global->filename () == 0 ? 0 : new UTL_String(idl_global->filename (), true)), // need a copy because set_filename() destroys previous value
```

Comment:
```suggestion
        : old_filename_ (idl_global->filename () ? new UTL_String (idl_global->filename (), true) : 0), 
        // need a copy because set_filename() destroys previous value
```
I didn't see a set_filename() in this class, so I guess this is IDL_GlobalData's set_filename?  Maybe clarify in the comment.